app/helpers/url_safe_serializer.py

```diff
@@ -6,4 +6,5 @@
 
 
 def url_safe_serializer() -> URLSafeSerializer:
-    return URLSafeSerializer(current_app.secret_key, salt=cookie_session[EQ_SESSION_ID])
+    # Type Ignore: Changing secret key at this point would be too late.
```

Comment:
This type ignore comment is confusing. The comment should be around the fact that we validate that it exists on app start up, hence it must exist at this point.